classes/book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'item'
+class Book < Item
+  attr_accessor :cover_state, :publisher, :published_date
+
+  def initialize(id, genre, author, source, label, cover_state, publisher,
+                 published_date)
```

Comment:
The `id` attribute is already generated from the parent class `item` so there is no need to pass it in when initializing the book subclass. so id suggests removing the id from the arguments list.